.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,41 +1,22 @@
-<!--
-Before submitting a pull request, please read:
-
-- the CONTRIBUTING guide at https://github.com/nodejs/node/blob/HEAD/CONTRIBUTING.md
-- the commit message formatting guidelines at
-  https://github.com/nodejs/node/blob/HEAD/doc/contributing/pull-requests.md#commit-message-guidelines
-
-For code changes:
-1. Include tests for any bug fixes or new features.
-2. Update documentation if relevant.
-3. Ensure that `make -j4 test` (UNIX), or `vcbuild test` (Windows) passes.
-
-If you believe this PR should be highlighted in the Node.js CHANGELOG
-please add the `notable-change` label.
+## Summary of Changes
 
-Developer's Certificate of Origin 1.1
+<!--
+Briefly describe the changes introduced by this pull request.
 
-By making a contribution to this project, I certify that:
+If your PR resolves an issue, please link it:
+Fixes #XXXX
+Related to #XXXX
+-->
 
-(a) The contribution was created in whole or in part by me and I
-    have the right to submit it under the open source license
-    indicated in the file; or
+## Code Changes Checklist
 
-(b) The contribution is based upon previous work that, to the best
-    of my knowledge, is covered under an appropriate open source
-    license and I have the right under that license to submit that
-    work with modifications, whether created in whole or in part
-    by me, under the same open source license (unless I am
-    permitted to submit under a different license), as indicated
-    in the file; or
+- [ ] Includes tests for bug fixes or new features.
+- [ ] Updates relevant documentation.
+- [ ] Passes linting (`make -j4 lint` for UNIX, `vcbuild lint` for Windows).
+- [ ] Passes testing (`make -j4 test` for UNIX, `vcbuild test` for Windows).
```

Comment:
My fear with these is that there are mnay changes that do not require tests or doc changes. For instance, a PR that only updates docs wouldn't have tests. I don't have a specific better suggestion but it might be good to say something like includes tests if needed, updates relevant docs if needed, etc.

Reply:
\<!--
 Not every checkbox will apply to your PR, but check off as much as you
 think applies.
-->?

Reply:
Yeah, I spotted that, I just think it's easy to completely overlook. Much like I've had feedback that folks haven't actually read the existing comment block. As I said, I don't have a specific better suggestion but I do have reservations.

Reply:
If they skip over it, they then complete redundant tests, so it's their loss. (I know that might come off as rude)